src/modules/chrome/helperMenu/HelperMenu.svelte

```diff
@@ -0,0 +1,122 @@
+<script>
+  import { onMount } from 'svelte';
+  import sendEvent from '../../analytics/sendEvent';
+  import draggable from '../../common/draggable';
+  import isFunction from '../../common/isFunction';
+  import querySelector from '../../common/querySelector';
+  import Modal from '../../modal/Modal.svelte';
+  import { playerIdUrl } from '../../support/constants';
+  import getValue from '../../system/getValue';
+  import functionLookup from './functionLookup';
+
+  const isFixed = getValue('keepHelperMenuOnScreen');
+  const isDraggable = getValue('draggableHelperMenu');
+
+  onMount(() => {
+    if (isDraggable) draggable(querySelector('#helperMenu'));
+  });
+
+  let showMenu = false;
+  function toggle() {
+    showMenu = !showMenu;
+  }
+
+  function sendHelperEvent(name) {
+    sendEvent('helperMenu', name);
+  }
+
+  function callModalFunction(name, fn) {
+    if (isFunction(fn)) {
+      sendHelperEvent(name);
+      fn();
+      toggle();
+    }
+  }
+</script>
+<button
+  class:helperMenuFixed={ isFixed }
+  class:helperMenuMove={ isDraggable }
+  type="button"
+  id="toggle"
+  on:click={ toggle }>
+  Helper Menu
+</button>
+<Modal visible={ showMenu } on:close={ toggle }>
+  <div id="helperMenu">
+    { #each functionLookup as menuSection }
+      <h2>{ menuSection.section }</h2>
+      <ul>{ #each menuSection.menu as menuItem }
+        <li>
+          { #if menuItem.fn }
+            <button type="button" on:click={ () => callModalFunction(menuItem.label, menuItem.fn) }>
+              { menuItem.label }
+            </button>
+          { :else if menuItem.href }
+            <a href={ menuItem.href } on:click={ () => sendHelperEvent(menuItem.label) }>
+              { menuItem.label }
+            </a>
+          { :else if menuItem.playerName }
+            <button
+              type="button"
+              class="helperDevBtn"
+              on:click={ () => { toggle(); sendHelperEvent('sendMsg'); } }
+              onclick="openQuickMsgDialog('{ menuItem.playerName }');">
```

Comment:
i particularly don't like inline js getting eval'ed. onclick shows up in the DOM whereas on:click doesn't.
there are 3 lines of code being executed on:click. i think this should be in a function.
alternatively you can have multiple on:click, might be ugly though.